### ISSUE TITLE
Idea: add wily to dev requirements and define configuration

### ISSUE THREAD

**Issue by redeboer** [ComPWA/expertsystem#46]
The ongoing refactoring efforts aim to make the code less complex and easier to contribute to. As this refactoring process will be a larger project consisting of several iterations, it may be worth to monitor the code complexity with [`wily`](https://wily.readthedocs.io/).

**Comment by spflueger** [comment]
Yeah, definitely an option, before we add it we should figure out how much it will actually help. DeepCode is pretty useless so far...

**Comment by redeboer** [comment]
DeepCode serves another purpose, I think. At least its output is different.

What I'm mainly interested in here is to see whether the refactoring steps reduce the complexity of the code and `wily` does its analysis for all commits in the master branch, so you have a nice way to see whether things are going the right direction.

**Comment by spflueger** [comment]
I'm simply talking about how reliable their output is

**Comment by redeboer** [comment]
Ah ok. Yeah, as in, DeepCode was a bit of a bummer as well...
